src/Plugins/DavisIMipPlugin.php

```diff
@@ -75,6 +75,15 @@ public function schedule(ITip\Message $itip)
         $senderEmail = substr($itip->sender, 7);
         $recipientEmail = substr($itip->recipient, 7);
 
+        if ($itip->senderName) {
+            $senderName = $itip->senderName.' <'.$senderEmail.'>';
+        } else {
+            $senderName = $senderEmail;
```

Comment:
I can understand why, but on most clients this will create a not very nice display since the header will be:

```
From: "email@email.com" <email@email.com>
```

Adding the email as the name doesn't bring much, I'd rather leave it empty (most clients will default to displaying the raw email)

Reply:
Unfortunately it isn't empty though as "(via Davis)" is appended regardless

Reply:
See comment above